docs/development.md

```diff
@@ -119,22 +119,28 @@ started prerequisites section](./getting-started.md#Prerequisites)
 
 #### Building and pushing dev images
 
-1. Login to your container registry using `docker login`.
-
-   e.g., `docker login quay.io`
-
-2. To build images with custom tags and push to your custom image registry,
+1. To build images with custom tags,
    run the `make docker-build` as follows:
 
    ```bash
-   REGISTRY="<container-registry>" MANAGER_IMAGE_TAG="<image-tag>" make docker-build
+   export REGISTRY="<container-registry>"
+   PULL_POLICY=IfNotPresent make docker-build
    ```
 
-3. Push your docker images:
+2. (optional) Push your docker images:
+
+   2.1. Login to your container registry using `docker login`.
+
+   e.g., `docker login quay.io`
+
+   2.2. Push to your custom image registry:
+
    ```bash
    REGISTRY="<container-registry>" MANAGER_IMAGE_TAG="<image-tag>" make docker-push
```

Comment:
Seems like the MANAGER_IMAGE_TAG is not mentioned above (removed) is it now required or is it missing from above?

Reply:
it's optional since it defaults to `dev`, added a comment above